native_client/client.cc

```diff
@@ -26,59 +26,39 @@
 
 using namespace DeepSpeech;
 
-struct ds_result {
+typedef struct {
```

Comment:
A class is just a struct where members are private by default. I don't see how either is more modern, but I made them classes anyway.

I don't use an std::string here because that makes it harder to handle the nullptr return case of stt.